### DISCUSSION TITLE
Increased refresh rate (Freestyle Libre 3)

### DISCUSSION THREAD

**Discussion by zibra** [NightscoutFoundation/xDrip#2675] [Ideas]
It will be great to increase refresh rate from 5 to 1 minute as I'm using Freestyle Libre 3.
Is it possible somehow already maybe?

**Comment by Navid200** [comment]
This has come up before.  The response from the developers has been negative.  For details please search.

**Comment by enpa23** [comment]
You could use this forked version of xDrip.
(link removed)
It comes however with other modifications which you may or may not like. You can find them listed in the home of the project.

**Reply by Navid200** [reply]
Please don't promote other forks.
We cannot stop people from creating forks.
But, xDrip is not a computer game.  It is effectively a medical app.  
Please stop posting links to other forks on this platform.

Everyone is welcome to be part of this team.  If you choose to go on your own, you can.  But, stop using our resources to promote your choice.

**Reply by enpa23** [reply]
Being this an open source project i think it's in its nature that people it's forking it. 
Also, I think that anyone who's coming to this github repo is enough informed about the risks of using modified apps and able to decide for itself, just as it did switching from the official abbt app to this one.

Anyway, I've removed the link as I understand this is not welcome.

**Reply by Navid200** [reply]
Everything you have stated is correct.  No one has done anything illegal.  I'm sorry if my post was abrupt.

This is my PR:  https://github.com/NightscoutFoundation/xDrip/pull/1759
It adds a feature that allows users to adjust the y axis range.  Many have asked for it, and I would like to have it too as I am also diabetic and use xDrip 24-7.
I can create my own version of xDrip and promote it and I am certain that many will start using it.
But, I believe that we (all developers) stand stronger together rather than alone as individuals.
So, I choose to stay here and continue to support this project (xDrip, the official xDrip) even though my personal preferences are sometimes rejected.

When someone posts in the facebook support group and asks for help, we cannot help them if they use an unofficial version.  It would make life much more difficult for those who support users if they had to deal with multiple versions of xDrip.
There are different reasons splitting the community is not going to help anyone if you look at the big picture.

Usually, if a PR is not accepted, there is a very good reason.  The way to proceed is to be patient and talk about it and try to understand the other angles and present it with the necessary modifications to address the shortcomings.  It will either get to a point that the developer proposing it understands why it is not a good idea, or it will eventually is accepted.

If every one of us creates our own version and distributes it, there will be no xDrip because the community will get divided and divided until there will be no way to know which is which.

**Comment by jamorham** [comment]
The feature is coming soon and I will put it out behind engineering mode first. I have been testing it and some things are broken but mostly it works due to recent changes.

**Comment by jamorham** [comment]
See detail of how to test this on issue ticket: https://github.com/NightscoutFoundation/xDrip/issues/2680
